src/lualib/InstanceOf.ts

```diff
@@ -1,10 +1,10 @@
 function __TS__InstanceOf(this: void, obj: LuaClassInstance, classTbl: LuaClass): boolean {
     if (typeof classTbl !== "object") {
-        // tslint:disable-next-line: no-string-throw
         throw "Right-hand side of 'instanceof' is not an object";
     }
 
     if (classTbl[Symbol.hasInstance] !== undefined) {
+        // eslint-disable-next-line no-implicit-coercion
         return !!classTbl[Symbol.hasInstance](obj);
```

Comment:
wouldnt it be nicer to just change this to an actual comparison

Reply:
A comparison with `true`? I'm fine with doing that for now, but in the future (when we'll implement boolean coercion algorithm) I think it should use `Boolean(...)`, since TypeScript doesn't check implementations of well-known symbols.